text-objects.kak

```diff
@@ -79,18 +75,22 @@ define-command -hidden text-object-indented-paragraph %{
   execute-keys '<a-i>i<a-z>i'
 }
 
-# depends on occivink/vertical-selection.kak
+# depends on occivink/kakoune-vertical-selection
 define-command -hidden text-object-vertical %{
-  evaluate-commands %sh{
-    case "$kak_opt_objects_last_mode" in
-      '<a-i>') k='<esc>:<space>select-vertically<ret>' ;;
-      '<a-a>') k='<a-i>w<esc>:<space>select-vertically<ret>' ;;
-      '[') k='<esc>:<space>select-up<ret>' ;;
-      ']') k='<esc>:<space>select-down<ret>' ;;
-      '{') k='<a-i>w<esc>:<space>select-up<ret>' ;;
-      '}') k='<a-i>w<esc>:<space>select-down<ret>' ;;
-    esac
-    [ -n "$k" ] && echo "execute-keys $k"
+  try %{
+    evaluate-commands %sh{
+      case "$kak_opt_objects_last_mode" in
+        '<a-i>') k='<esc>:<space>select-vertically<ret>' ;;
+        '<a-a>') k='<a-i>w<esc>:<space>select-vertically<ret>' ;;
+        '[') k='<esc>:<space>select-up<ret>' ;;
+        ']') k='<esc>:<space>select-down<ret>' ;;
+        '{') k='<a-i>w<esc>:<space>select-up<ret>' ;;
+        '}') k='<a-i>w<esc>:<space>select-down<ret>' ;;
+      esac
+      [ -n "$k" ] && echo "execute-keys $k"
+    }
+  } catch %{
+    fail "no selections remaining" 
```

Comment:
Previously I've made try catch block show warning if plugin wasn't installed, that external vertical selection plugin is required. However when plugin is installed, but it isn't able to select anything, the same warning is shown. Therefore I've changed it to "no selections remaining" error. Kakoune defaults to this message if there's no selection remaining after any operation.